src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -242,6 +248,20 @@
       <FileWrites Include="$(_GeneratedProfileClass)" />
     </ItemGroup>
   </Target>
+
+  <Target Name="_GenerateTypeMapGeneratorBCLAssemblyNamesClass"
+      BeforeTargets="CoreCompile"
+      Inputs="@(_TypeMapGeneraterBCLAssemblyName)"
+      Outputs="$(_GeneratedTypeMapGeneratorBCLAssemblyNamesClass)">
+    <GenerateTypeMapGeneratorBCLAssemblyNames Files="@(_TypeMapGeneraterBCLAssemblyName)" OutputFile="$(_GeneratedTypeMapGeneratorBCLAssemblyNamesClass)" />
+    <WriteLinesToFile
+        File="$(IntermediateOutputPath)$(CleanFile)"
+        Lines="$(_GeneratedTypeMapGeneratorBCLAssemblyNamesClass)"
+        Overwrite="false"/>
+    <ItemGroup>
+      <FileWrites Include="$(_GeneratedTypeMapGeneratorBCLAssemblyNamesClass)" />
+    </ItemGroup>
+  </Target>
```

Comment:
Can we generate this file ahead of time? Put it in the shared runtime? Or just ship it on disk?

Reply:
This is actually not needed anymore, it was a remnant from one of the earlier iterations. Removed